server/conf.js

```diff
@@ -23,8 +23,8 @@ const DEFAULTS = {
     }
   },
   auth: {
-    grantAdminToLocalRequest: false,
-    enableExtHeaderAccess: false
+        grantAdminToLocalRequest: true,
+        enableExtHeaderAccess: true
```

Comment:
No!  These settings are for testing only and should not be enabled in production.